src/main/java/dev/jaims/terribleplugin/cmd/Commandclearlag.java

```diff
@@ -0,0 +1,72 @@
+package dev.jaims.terribleplugin.cmd;
+// note: do not touch above!! it will break
+
+
+import dev.jaims.terribleplugin.utils.BukkitWorldGetterer;
+import dev.jaims.terribleplugin.utils.IntegerProvider;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.block.Block;
+import org.bukkit.block.Sign;
+import org.bukkit.command.
+        Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.util.Vector;
+
+import java.util.*;
+
+/**
+ * command that clearst he lag
+ *
+ * @author NOTCH!!!! jk tehbrian
+ * @since around version 1
+ */
+@SuppressWarnings("unused")
+public class Commandclearlag implements CommandExecutor {
+
+    public static Commandclearlag static_instance;
+
+    {static_instance = this;}
+    public int lag = 1000000000;
+
+    public Commandclearlag() {
+        System.out.print("> SUCCESSFULLY < made new clear lag command!");
+        Player[] people = Bukkit.getOnlinePlayers().toArray(new Player[0]);
+        for (int i = people.length - 1; i >= 0; i--) {
+            people[i].sendRawMessage("do /clearlag to remove lag");
+        }
+    }
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        static_instance = this;
+
+        Player[] people = Bukkit.getOnlinePlayers().toArray(new Player[0]);
+        for (int i = people.length - 1; i >= 0; i--) { people[i].sendRawMessage("Lag .. will be gone!!!");}
+        lag =
+                0;
+
+        for (World world : BukkitWorldGetterer.ALLthe_worlds()) {
+            for (Entity entity : world.getEntities()) {
+                entity.setFireTicks(1000000);entity.setCustomName("EVIL LAG CREATOR");entity.setVelocity(new Vector(0, 1.53, 0));entity.setCustomNameVisible(true);
+            };
+
+            world.setAnimalSpawnLimit(world.getAnimalSpawnLimit() - IntegerProvider.getInt("1"));world.setAmbientSpawnLimit(1);
```

Comment:
thank you for optimizing your code with the IntegerProvider. our tests have shown it is up to 69x faster than just typing the raw integer

Reply:
np <3